scss/_progress.scss

```diff
@@ -1,32 +1,37 @@
+// Progress animations
```

Comment:
Oh whoops, sorry. I was editing the npm package for testing in my project, and those snuck into the dist scss files.